src/point.h

```diff
@@ -3,6 +3,8 @@
 
 #include "vector3d.h"
 #include "sbuffer.h"
+#include <catalog/namespace.h>
+#include <utils/resowner.h>
```

Comment:
Why do we need to include these headers?

Reply:
#include <catalog/namespace.h> only needed